packages/extension/src/newtab/App.tsx

```diff
@@ -134,12 +141,29 @@ function InternalApp(): ReactElement {
     }
   }, [contentScriptGranted]);
 
+  useEffect(() => {
+    if (!user || shouldShowLogin) {
```

Comment:
Why do you have this condition?

I'm not sure it's needed, we need to show the overlay on the first tab regardless if the user is logged in or not

Reply:
Right. Our flow is install -> registration -> new tab. But yeah, if the user skips installation, we should still show this. Let me remove it.

Reply:
Can you show a video where the user doesn't signup and we still show the overlay please?